### PR TITLE
Madelyn's attempt as a logo

### PR DESCRIPTION
**Pull request by madolson**
My attempt at a logo for https://github.com/valkey-io/valkey/issues/82#issuecomment-2035079091.

It is one character wider and 2 characters taller than the original Redis logo.

### CODE REVIEW

File: src/asciilogo.h [hunk 1]
```diff
@@ -28,20 +28,22 @@
...
+" |###|   |###(  (@@@)  )###|   |###|          https://valkey.io      \n"
+" |###|   |####.  '-'  .####|   |###|                                 \n"
+" |###|   |#####*.   .*#####|   |###|                                 \n"
+" |###|   '+#####|   |#####+'   |###|                                 \n"
+" |####+.     +##|   |#+'     .+####|                                 \n"
+" '#######+   |##|        .+########'                                 \n"
+"    '+###|   |##|    .+########+'                                    \n"
+"        '|   |####+########+'                                        \n"
+"             +#########+'                                            \n"
+"                '+v+'                                                \n\n";
```
Comment (zuiderkwast) [inline src/asciilogo.h]: The center dot is "@@@". Is that intentional?

```suggestion
"                .+^+.                                                \n"
"            .+#########+.                                            \n"
"        .+########+########+.           Valkey %s (%s/%d) %s bit\n"
"    .+########+'     '+########+.                                    \n"
" .########+'     .+.     '+########.    Running in %s mode\n"
" |####+'     .+#######+.     '+####|    Port: %d\n"
" |###|   .+###############+.   |###|                                 \n"
" |###|   |#####*'' ''*#####|   |###|                                 \n"
" |###|   |####'  .-.  '####|   |###|                                 \n"
" |###|   |###(  (###)  )###|   |###|          https://valkey.io      \n"
" |###|   |####.  '-'  .####|   |###|                                 \n"
" |###|   |#####*.   .*#####|   |###|                                 \n"
" |###|   '+#####|   |#####+'   |###|                                 \n"
" |####+.     +##|   |#+'     .+####|                                 \n"
" '#######+   |##|        .+########'                                 \n"
"    '+###|   |##|    .+########+'                                    \n"
"        '|   |####+########+'                                        \n"
"             +#########+'                                            \n"
"                '+v+'                                                \n\n";
```

**Review by zuiderkwast — APPROVED** [review]

File: src/asciilogo.h (hunk 1 above)
Comment (madolson) [reply src/asciilogo.h]: It was! Because the center is circle I wanted circle vibes. Don't feel strongly though

Comment (zuiderkwast) [reply src/asciilogo.h]: Leave it then. :) I thought it was unintentional.

**Comment by zuiderkwast** [comment]
Did you delete `PID: %ld` next to the logo?

The test case fails because of that.

**Comment by madolson** [comment]
> The test case fails because of that.

What an interesting dependency.

**Comment by madolson** [comment]
Given this is easy to revert, I'm going to optimistically push it and we can revert it.

**Pull request merged**